cognite_toolkit/_cdf_tk/prototypes/commands/dump_timeseries.py

```diff
@@ -0,0 +1,342 @@
+from __future__ import annotations
+
+import shutil
+from collections.abc import Iterator
+from functools import lru_cache
+from pathlib import Path
+from typing import Any, Literal, cast
+
+import pandas as pd
+import questionary
+import yaml
+from cognite.client import CogniteClient
+from cognite.client.data_classes import (
+    DataSetWrite,
+    DataSetWriteList,
+    TimeSeriesFilter,
+    TimeSeriesList,
+)
+from cognite.client.data_classes.filters import Equals
+from cognite.client.exceptions import CogniteAPIError
+from rich.progress import Progress, TaskID
+
+from cognite_toolkit._cdf_tk.commands._base import ToolkitCommand
+from cognite_toolkit._cdf_tk.exceptions import (
+    ToolkitFileExistsError,
+    ToolkitIsADirectoryError,
+    ToolkitMissingResourceError,
+    ToolkitValueError,
+)
+from cognite_toolkit._cdf_tk.loaders import DataSetsLoader
+from cognite_toolkit._cdf_tk.utils import CDFToolConfig
+
+# from cognite_toolkit._cdf_tk.prototypes.resource_loaders import TimeSeriesLoader
+TIME_SERIES_FOLDER_NAME = "timeseries"
```

Comment:
Suggest just getting the folder name from the TimeSeriesLoader in the core of the code base.
```suggestion
from cognite_toolkit._cdf_tk.loaders import TimeSeriesLoader
TIME_SERIES_FOLDER_NAME = TimeSeriesLoader.folder_name
```